.github/workflows/simorgh-create-github-release.yml

```diff
@@ -0,0 +1,34 @@
+name: Simorgh CD - Create Release Tag
+
+on:
+  push:
+    branches:
+      - 'latest'
+
+jobs:
+  build:
+    name: Create Release Tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Bump version and push tag
+        id: release_tag
+        uses: anothrNick/github-tag-action@1.26.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          RELEASE_BRANCHES: 'latest'
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ steps.release_tag.outputs.new_tag }}
+          release_name: Release ${{ steps.release_tag.outputs.new_tag }}
+          body: |
+            Version: ${{ steps.release_tag.outputs.new_tag }}
```

Comment:
I have been experimenting with release notes in this PR https://github.com/bbc/simorgh/pull/8279

Wondering if we could remove version (since it's already in the release name and tag name) and just add the text of the last commit, which will contain PR details

```suggestion
            ${{ github.event.commits[github.event.commits.distinct_size].message }}
```

I spoofed a merge commit and this is what was generated: https://github.com/bbc/simorgh/releases/tag/untagged-30330902de830e30934c

It's also similar to what we see when we tag a release
![image](https://user-images.githubusercontent.com/58214768/97741964-849d6680-1adb-11eb-9231-0e2fa94843af.png)

Reply:
For further context, here's the docs on the github.event for a PushEvent https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/github-event-types#pushevent